app/bundles/LeadBundle/Model/CompanyModel.php

```diff
@@ -920,11 +921,7 @@ public function checkForDuplicateCompanies(array $queryFields): array
         return $this->companyDeduper->checkForDuplicateCompanies($queryFields);
     }
 
-    /**
-     * @param array $fields
-     * @param array $data
-     */
-    protected function getFieldData($fields, $data): array
+    protected function getFieldData(array $fields, array $data): array
```

Comment:
Is this change necessary? Changing a protected method signature is considered a BC break as plugins that extend this method will break.

I'm not that touchy-feely about this as this should have been a private method from the beginning. But if there is no reason for this change, please revert it. Or rebase to the 7.x branch which can have BC breaks.